lms/envs/aws.py

```diff
@@ -445,6 +454,18 @@
 for database_name in DATABASES:
     DATABASES[database_name]['ATOMIC_REQUESTS'] = True
 
+# Override database settings with environment variables, if any
+for name, database in DATABASES.items():
+    if name != 'read_replica':
+        database.update({
+            'ENGINE': os.environ.get('DB_ENGINE', database['ENGINE']),
```

Comment:
My reasoning was that by moving the database setting override out of the
aws_migrate settings file into the main settings file, the override is no
longer migration-specific: you can now always override the database
settings with environment variables. Hence the name change